joomla/joomla-cms
# Removed "readonly" and "disabled" fields parameters

Pull Request for Issue #15369 .

### Summary of Changes
This PR proposes to remove the "disabled" and "readonly" from the fields parameters as they don't make any sense for a field.

### Testing Instructions
* Create a new field or save an existing one again so the readonly/disabled parameters aren't stored anymore.
* Test that the field works as expected.
* Test especially also that ACL restrictions still work, eg a user which isn't allowed to edit a value has a readonly/disabeld field.

### Documentation Changes Required
Maybe if documentation is already written with those parameters mentioned.
